reporter/src/funTest/assets/spdx-document-reporter-expected-output.spdx.json

```diff
@@ -0,0 +1,14 @@
+{
+  "SPDXID" : "SPDXRef-<REPLACE_UUID>",
```

Comment:
The `SPDXID` for the document itself should always be the literal text "SPDXRef-DOCUMENT" -- see https://spdx.github.io/spdx-spec/v2-draft/document-creation-information/#63-spdx-identifier-field

Reply:
Thanks @swinslow, greatly appreciated!
I'll merge the PR as is now, and then address your comment(s) in a follow up PR.

Reply:
@swinslow - I've addressed your findings with #2812 .

Reply:
LGTM, thanks @fviernau!